pkg/miniogw/config.go

```diff
@@ -35,6 +35,14 @@ type RSConfig struct {
 	MaxThreshold     int `help:"the largest amount of pieces to encode to. n." default:"50"`
 }
 
+// EncryptionConfig is a configuration struct that keeps details about
+// encrypting segments
+type EncryptionConfig struct {
+	EncKey       string `help:"root key for encrypting the data" default:"supersecretpassword"`
```

Comment:
How about doing some PR in the default value of the encryption key: "highlydistributedridiculouslyresilient"?

Reply:
What do you mean "doing some PR"? I could change it now

Reply:
PR as "public relationship"/"marketing", not as "pull request".

Reply:
Ah, ok haha. I'll make the change